tensorflow/compiler/mlir/tosa/transforms/legalize_tfl.cc

```diff
@@ -1618,6 +1619,23 @@ LogicalResult ConvertTFLTransposeConvOp::matchAndRewrite(
   bool output_is_qtype =
       output_type.getElementType().isa<mlir::quant::QuantizedType>();
 
+  const bool has_bias = !(!tfl_conv_op.getBias() ||
+                          tfl_conv_op.getBias().getType().isa<NoneType>());
+
+  if (has_bias) {
+    RankedTensorType bias_type =
+        dyn_cast<RankedTensorType>(tfl_conv_op.getBias().getType());
+    bool bias_is_qtype =
```

Comment:
Prefer to use free functions here (isa<...>(X) instead of X.isa<...>), see MLIR deprecation page.

Reply:
Done, free functions are used across the whole function for consistency